src/views-components/process-input-dialog/process-input-dialog.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from "react";
+import { Dialog, DialogActions, Button, CardHeader, DialogContent } from '@material-ui/core';
+import { WithDialogProps } from '~/store/dialog/with-dialog';
+import { withDialog } from "~/store/dialog/with-dialog";
+import { PROCESS_INPUT_DIALOG_NAME } from '~/store/processes/process-input-actions';
+import { RunProcessInputsForm } from "~/views/run-process-panel/run-process-inputs-form";
+
+export const ProcessInputDialog = withDialog(PROCESS_INPUT_DIALOG_NAME)(
+    (props: WithDialogProps<any>) =>
+        <Dialog
+            open={props.open}
+            maxWidth={false}
+            onClose={props.closeDialog}>
+            <CardHeader
+                title="Inputs - Pipeline template that generates a config file from a template" />
+            <DialogContent>
+                <RunProcessInputsForm inputs={getInputs(props.data.containerRequest)} />
+            </DialogContent>
+            <DialogActions>
+                <Button
+                    variant='flat'
+                    color='primary'
+                    onClick={props.closeDialog}>
+                    Close
+                </Button>
+            </DialogActions>
+        </Dialog>
+);
+
+const getInputs = (data: any) =>
+    data && data.mounts.varLibCwlWorkflowJson ? data.mounts.varLibCwlWorkflowJson.content.graph[1].inputs.map((it: any) => (
```

Comment:
Maybe instead of `graph[1]` it would be safer to use `getWorkflowInputs` ?

Reply:
That's why I wanted u to check this PR :) :+1: